knative-operator/pkg/common/util.go

```diff
@@ -75,3 +77,28 @@ func SetOwnerAnnotations(instance *servingv1alpha1.KnativeServing) mf.Transforme
 		return nil
 	}
 }
+
+func EnsureContainerMemoryLimit(s *operatorv1alpha1.CommonSpec, containerName string, memory resource.Quantity) error {
```

Comment:
wanna add a test to `util_test.go`  ?

Reply:
yeah, thought about it, chose not to since the whole thing is tested twice in here:
- https://github.com/openshift-knative/serverless-operator/blob/66f67eb14211344e6ae13cef4fc06e1fba1efcf6/knative-operator/pkg/common/eventing_test.go#L46
- https://github.com/openshift-knative/serverless-operator/blob/66f67eb14211344e6ae13cef4fc06e1fba1efcf6/knative-operator/pkg/common/serving_test.go#L115

Reply:
sounds good to me

/lgtm